### DISCUSSION TITLE
What is select mode in mappings?

### DISCUSSION THREAD

**Discussion by edshamis** [hrsh7th/nvim-cmp#1815] [Q&A] [answered]
Could someone shed some light on it?

**Comment by Shougo** [comment, marked as answer]
Please see `:help select-mode`.

It is mainly used for snippet plugins.

**Comment by edshamis** [comment]
Maybe you have an example where it is used?

**Reply by Shougo** [reply]
For example:

https://github.com/hrsh7th/vim-vsnip/blob/02a8e79295c9733434aab4e0e2b8c4b7cea9f3a9/README.md?plain=1#L67
